UKHomeOffice/nmsw-ui
# NMSW-20 Add link to Crown Copyright Logo in footer

# Ticket

NMSW-20

----

## AC

GIVEN I am viewing a page
WHEN I click on the Crown Copyright logo in the footer
THEN I am taken to the national archives crown copyright page in a new tab

----

## To test

- run app locally
- go to localhost:3000
- go to the footer
- click on the crown copyright logo
- > you should be taken to a new tab with: https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/

----

## Notes

- index.html already contains `<meta name="robots" content="noindex nofollow" />`
